ha/fault_tolerance/event_manager.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python3
+
+'''Module whoch provides interfaces to subscribe or unsubscribe for HA
+   events to cortx components'''
+
+import json
+
+from cortx.utils.log import Log
+from ha.core.config.config_manager import ConfigManager
+
+
+class EventManager:
```

Comment:
The LLD also has another class "SubscribeEvent" to get the event_list.
Is that part of this ticket or some other ticket ? I did not see it in EOS-17962 also.
In case that class is no longer needed, can we update the LLD ? @indrajitzagade  @Madhura-08

Reply:
Not sure about this.

Reply:
@Madhura-08 We will discuss about "SubscribeEvent" class. This class is needed.

Reply:
@Madhura-08 As discussed lets update subscribe and unsubscribe methods that would accept list of "SubscribeEvent" objects as events list.

Reply:
Done

Reply:
>as discussed lets update subscribe and unsubscribe methods that would accept list of "SubscribeEvent" objects as events list.
I think this is not handled yet